pupil-spa/src/app/user.data.mock.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
if you look in `/design/spa/checksession.json` you will find the actual spec for the user data.  Probably best to replace it as many of the fields below do not exist in our dataset